packages/effect/src/Schema.ts

```diff
@@ -5782,6 +5782,42 @@ export const StringFromHex: Schema<string> = makeEncodingTransformation(
   Encoding.encodeHex
 )
 
+/**
+ * Decodes a URI component encoded string into a UTF-8 string.
```

Comment:
I'll add this tip to the JSDoc description:

> Very useful for storing data in the url with a Schema : 

I will also include this example you provided in the PR description, as it's quite helpful:

```ts
import { Schema } from "effect"

const PaginationSchema = Schema.Struct({
  maxItemPerPage: Schema.Number,
  page: Schema.Number
})

const UrlSchema = Schema.compose(Schema.StringFromUriComponent, Schema.parseJson(PaginationSchema))

console.log(Schema.encodeSync(UrlSchema)({ maxItemPerPage: 10, page: 1 }))
// Output: TODO: the actual output
```

Reply:
Thanks for the review, I have changed the description accordingly